### PR TITLE
[rest] VoiceResource: Return answer from /interpreters endpoint

### PR DESCRIPTION
**Pull request by florian-h05**
Also add annotations for the response.

### CODE REVIEW

**Comment by florian-h05** [comment]
As the current answer was undocumented, WDYT about changing the format to JSON and putting the current answer string into a answer key?
My plans for the UI need extended core APIs ...

**Comment by wborn** [comment]
> WDYT about changing the format to JSON and putting the current answer string into a answer key?

I wouldn't change it now unless there is a good reason and it is actually used for that. Seems like this API has been around  since OH 2.x so changing it will most likely may break some user integrations.

File: bundles/org.openhab.core.io.rest.voice/src/main/java/org/openhab/core/io/rest/voice/internal/VoiceResource.java [hunk 1]
```diff
@@ -189,8 +191,8 @@ public Response interpret(
         }
 
         try {
-            hli.interpret(locale, text);
-            return Response.ok(null, MediaType.TEXT_PLAIN).build();
+            String answer = hli.interpret(locale, text);
+            return Response.ok(answer, MediaType.TEXT_PLAIN).build();
```
Comment (wborn) [inline bundles/org.openhab.core.io.rest.voice/src/main/java/org/openhab/core/io/rest/voice/internal/VoiceResource.java:195]: Did it always return null?

**Comment by florian-h05** [comment]
My problem is, that I not only need to return an answer from the HLI but also some „action“ data to instruct the UI to show a widget for some Item, but that would also need changes to the HLI interface. Not sure how to handle that.

File: bundles/org.openhab.core.io.rest.voice/src/main/java/org/openhab/core/io/rest/voice/internal/VoiceResource.java (hunk 1 above)
Comment (florian-h05) [reply bundles/org.openhab.core.io.rest.voice/src/main/java/org/openhab/core/io/rest/voice/internal/VoiceResource.java:195]: What to you mean?

**Comment by wborn** [comment]
It might be possible to support both if you add another method with the `@Consumes(MediaType.APPLICATION_JSON)` annotation and keep the method having `@Consumes(MediaType.TEXT_PLAIN)`. :wink:

**Comment by florian-h05** [comment]
Thanks for the hint — I will try that out then, but let’s get this merged first.

**Review by wborn — APPROVED** [review]
Thanks!

**Pull request merged**